### ISSUE TITLE
Broken undelegated test

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#779]
GUI sends the following request with no nameserver IP address and is accepted (no issue):
```json
{
  "jsonrpc": "2.0",
  "id": 1622401520693,
  "method": "start_domain_test",
  "params": {
    "ipv4": true,
    "ipv6": true,
    "profile": "default",
    "domain": "iis.se",
    "ds_info": [],
    "nameservers": [
      {
        "ns": "ns.nic.se",
        "ip": ""
      }
    ],
    "client_version": "3.2.2",
    "client_id": "Zonemaster GUI"
  }
}
```

GUI send the following request with nameserver IP address set:
```json
{
  "jsonrpc": "2.0",
  "id": 1622401208379,
  "method": "start_domain_test",
  "params": {
    "ipv4": true,
    "ipv6": true,
    "profile": "default",
    "domain": "iis.se",
    "ds_info": [],
    "nameservers": [
      {
        "ns": "ns.nic.se",
        "ip": "8.8.8.8"
      }
    ],
    "client_version": "3.2.2",
    "client_id": "Zonemaster GUI"
  }
}
```
and gets the response 

```json
{
  "jsonrpc": "2.0",
  "error": {
    "data": "/nameservers/0/ip: String does not match (?^u:^$|(?^u:^[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\$)|(?^ui:^([0-9a-f]{1,4}:[0-9a-f:]{1,}(:[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3})?)\\$|([0-9a-f]{1,4}::[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3})\\$)).",
    "code": "-32602",
    "message": "Invalid method parameter(s)."
  },
  "id": 1622401208379
}
```
The issue is the same if `zmb` is used.

**Comment by matsduf** [comment]
Will be resolved by #782.

**Comment by mattias-p** [comment]
Fixed by #782.
